package.json

```diff
@@ -1,8 +1,8 @@
 {
   "author": "Elamaran Shanmugam @elamaras",
   "license": "MIT",
-  "name": "aws-eks-3pa",
-  "description": "AWS EKS 3PA Transformer is a solution that provides pre-launch validations of the partner software on compatibility based on the marketplace guidelines, covering static and dynamic (deployment/runtime) aspects.",
+  "name": "addons-transformer-for-amazon-eks",
+  "description": "Sleek Marketplace validator is solution provide pre-launch validations of the partner software on compatibility with Sleek guidelines, covering static and dynamic (deployment/runtime) aspects.",
```

Comment:
Remove sleek referenes here!

Reply:
Like `Addons Transformer for Amazon EKS is a solution that provides pre-launch validations of the partner software based on`